doc/Language/grammars.pod6

```diff
@@ -288,7 +288,7 @@ the number for the currently parsed C<digit> digits.
     say Digifier.parse('255 435 777', actions => Devanagari.new).made;
     # OUTPUT: «(२५५ ४३५ ७७७)␤»
 
-=head2 Methods in Grammar
+=head2 Methods in Grammars
```

Comment:
Can you please not use capitals here? Either you use C<Grammar>s or simply grammars.